ert/storage/_storage.py

```diff
@@ -544,6 +544,15 @@ def init(*, workspace_name: str) -> None:
         )
 
 
+def assert_storage_initialized(workspace_name: str) -> None:
+    response = _get_from_server(path="experiments")
+    experiment_names = {exp["name"]: exp["ensemble_ids"] for exp in response.json()}
+
+    for special_key in _SPECIAL_KEYS:
+        if f"{workspace_name}.{special_key}" not in experiment_names:
+            raise ert.exceptions.StorageError("Storage has not been initialized")
```

Comment:
Can we give a hint as to how the user can solve this? Maybe something like: "Storage has not been initialized with this workspace. To fix this, the workspace need to be initialized again"

Reply:
I have update the message. For this particular case, where `ert3 init` failed, the user should not see it anymore. If he/she does see it, then `ert3 init` was fine, but something else is broken with the storage and it needs to be re-initialized. That is currently not possible, since that requires re-running `ert3 init`, which will fail since the `.ert` directory will be present. The solution is currently to remove the `.ert` directory and re-run `ert3 init`. So that needs some thought at some point.